packages/core/src/fetchers/fetchReferencedEntities.ts

```diff
@@ -0,0 +1,64 @@
+import { ExperienceEntry } from '@/types';
+import { ContentfulClientApi, Entry, Asset } from 'contentful';
+import { isExperienceEntry } from '@/utils';
+
+type fetchReferencedEntitiesArgs = {
```

Comment:
[nitpick]
Types are always written with a capital first letter (at least at Contentful). This is especially nice to consider as this is a public repo and the types are consumed by SDK users.
```suggestion
type FetchReferencedEntitiesArgs = {
```

Reply:
Same feedback for `fetchExperienceEntryArgs`, `fetchExperienceArgs`, etc.